SIP-X.md

```diff
@@ -0,0 +1,75 @@
+|                   |	LoanSettings                | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	ProtocolSettings            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	SwapsExternal               | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+| Other             |                             |       |                                                              |
+|                   |	Liquidity Mining            | Owner	| GovernorOwner (`0x6496df39d000478a7a7352c01e0e713835051ccd`) |
+|                   |	StakingRewards              | Owner	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	VestingLogic                | Owner	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Modules                     | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Connectors        |                             |       |                                                              |
+|                   |	LoanToken                   | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenLogicBeacon        | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenSettingsLowerAdmin | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	LoanTokenLogicStandard      | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Core              |	                            |     	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Protocol                    | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Oracles           |	                            |     	|                                                              |
+|                   |	BPro Price Feed             | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	RSK Price Feed              | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	MoC Price Feed              | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	Price Feeds Gateway         | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   |	PriceFeedV1PoolOracle       | Admin	| GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+| Governance        |	                            |      	|                                                              |
+|                   | SOV                         | Admin | GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
+|                   | Locked SOV                  | Admin | GovernorAdmin (`0xff25f66b7d7f385503d70574ae0170b6b1622dad`) |
```

Comment:
that's fine, but this contract can have multiple admins. do we suggest here to merley add the GovernorAdmin as an admin or replace exchequer?
It would be fine to replace it in my opinion. The admin can change the cliff and duration used by the LockedSOV contract, add and remove more admins and migrate tokens to a new contract if required (the last 3 should have rather been placed under owner imo, but it is what it is)

Reply:
> this contract can have multiple admins.

do you mean multiple admins who all have the same capabilities? or multiple admins who each have different capabilities?

> do we suggest here to merley add the GovernorAdmin as an admin or replace exchequer?

Replace Exchequer, if it makes sense to you given the use/power of this contract and what we are trying to accomplish with this transfer of governance from Exchequer to Bitocracy. Btw - what exactly does the Locked SOV contract do? :)

> (the last 3 should have rather been placed under owner imo, but it is what it is)

Would it require a new contract to do this? Would you prefer that we change the `Admin` role to be held by GovernorOwner instead of GovernorAdmin in this case?

Reply:
Answered all of the above.

- All admins have the same capabilities
- Locked SOV is an "intermediate" contract between rewards contracts and the staking/vesting contract, after rewards are earned but before they are claimed/withdrawn by the user.
- Admin role will be assigned to GovernorAdmin for now.